backend/src/database/migrations/20221201094044_create-token.js

```diff
@@ -0,0 +1,12 @@
+export async function up(knex) {
+    return knex.schema.createTable("tokens", (table) => {
+        table.increments("id").primary();
```

Comment:
We are using UUID not incremental id's. Make this change.

Reply:
Fixed.